docs/k0s-in-docker.md

```diff
@@ -0,0 +1,31 @@
+# k0s in Docker
+
+We publish k0s container image for every release. By default we run both controller and worker in the same container to allow easy local test "cluster".
+
+You can run your own k0s-in-docker easily with:
+```sh
+docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.pkg.github.com/k0sproject/k0s/k0s:<version>
```

Comment:
The image is not publicly available:
```
$ docker run -d --name k0s --hostname k0s --privileged -v /var/lib/k0s -p 6443:6443 docker.pkg.github.com/k0sproject/k0s/k0s:latest
Unable to find image 'docker.pkg.github.com/k0sproject/k0s/k0s:latest' locally
docker: Error response from daemon: Get https://docker.pkg.github.com/v2/k0sproject/k0s/k0s/manifests/latest: no basic auth credentials.
```

Reply:
I know, but it will be after the next release